### DISCUSSION TITLE
Log Retention in SO 2.4

### DISCUSSION THREAD

**Discussion by loganterfehr** [Security-Onion-Solutions/securityonion#12190] [Unsupported Versions]
If I wanted to delete out old logs after 90 days how would I go about doing that? I want to set a retention policy of 90 days of stored logs and then after that period delete the oldest out. I noticed that SO no longer uses Curator, so how would I go about this? Thanks. Any help is appreciated.

**Comment by cm-ops** [comment]
If you wanted to do all logs you could set the delete policy in `Administration > configuration > elasticsearch >index_settings > index_name > policy > phases > delete` You will need advanced toggled on and set it to `90d`.

**Reply by loganterfehr** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/122332360/659fb753-9b56-4eb8-93b4-86149c872ec7)

Do you mean elasticsearch > index_settings > index_template > phases > delete > min_age set to 90?

I do not have index_name or policy

**Reply by cm-ops** [reply]
You would need to select the index name, for example `so-zeek`

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/63416267/d6b2c76c-67f0-4082-b58d-500181bf67d1)

**Reply by loganterfehr** [reply]
And do this for each index if I wanted to have all logs retained for a max of 90 days?

**Reply by loganterfehr** [reply]
Or would the 'global_overrides' section do it for all? It seems like it would.
